src/main/resources/templates/template_offender-case-notes.mustache

```diff
@@ -5,7 +5,7 @@
         <tr><td>Type</td><td>{{ optionalValue type }}</td></tr>
         <tr><td>Sub type</td><td>{{ optionalValue subType }}</td></tr>
         <tr><td>Creation date time</td><td>{{ optionalValue (formatDate creationDateTime) }}</td></tr>
-        <tr><td>Author name</td><td>{{ optionalValue authorName }}</td></tr>
+        <tr><td>Author name</td><td>{{ getUserLastName authorName }}</td></tr>
```

Comment:
Currently we don't actually get the username, but once it does start being sent through this will ensure it get's converted to the surname